### PR TITLE
Add `$project` and `$unset` to aggregation stages section

### LINKED ISSUE #3428: Add `$project` and `$unset` to documentation
### What should be done?

`$project` and `$unset` are currently [supported commands](https://docs.ferretdb.io/reference/supported-commands/#aggregation-pipeline-stages), and this should be reflected here https://docs.ferretdb.io/aggregation-operations/aggregation-stages/.

### PR DESCRIPTION
**Pull request by Akhaled19**
Added `$project` and `$unset` commands to the Aggregation Stages list and gave them a description.

Closes #3428.

### CODE REVIEW

**Comment by AlekSi** [comment]
@Akhaled19 please click CLA button in [this](https://github.com/FerretDB/FerretDB/pull/3450#issuecomment-1738416164) comment

File: website/docs/aggregation-operations/aggregation-stages.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ Aggregation stages are a series of one or more processes in a pipeline that acts
...
 | `$limit`                     | Limits specific documents and passes the rest to the next stage                                       |
 | `$match`                     | Acts as a `find` operation by only returning documents that match a specified query to the next stage |
+| `$project` | Specifies `n` number of fields in a document to pass to the next stage in the pipeline                               |
 | `$skip`                      | Skips a specified `n` number of documents and passes the rest to the next stage                       |
 | `$sort`                      | Sorts and returns all the documents based on a specified order                                        |
-| `$unwind`                    | Deconstructs and returns a document for every element in an array field                               |
+| `$unset` | Specifies `n` number of fields to be removed/excluded from a document. `$unset` is an alias for the `$project` stage         | 
+| `$unwind`                    | Deconstructs and returns a document for every element in an array field                               |
```
Comment (Fashander) [inline website/docs/aggregation-operations/aggregation-stages.md:19]:
```suggestion
| `$unwind`                    | Deconstructs and returns a document for every element in an array field                               |

```

File: website/docs/aggregation-operations/aggregation-stages.md [hunk 2]
```diff
@@ -12,6 +12,8 @@ Aggregation stages are a series of one or more processes in a pipeline that acts
 | `$group`                     | Groups documents based on specific value or expression and returns a single document for each group   |
 | `$limit`                     | Limits specific documents and passes the rest to the next stage                                       |
 | `$match`                     | Acts as a `find` operation by only returning documents that match a specified query to the next stage |
+| `$project` | Specifies `n` number of fields in a document to pass to the next stage in the pipeline                               |
 | `$skip`                      | Skips a specified `n` number of documents and passes the rest to the next stage                       |
 | `$sort`                      | Sorts and returns all the documents based on a specified order                                        |
-| `$unwind`                    | Deconstructs and returns a document for every element in an array field                               |
+| `$unset` | Specifies `n` number of fields to be removed/excluded from a document. `$unset` is an alias for the `$project` stage         |
```
Comment (Fashander) [inline website/docs/aggregation-operations/aggregation-stages.md:18]:
```suggestion
| `$unset` | Specifies `n` number of fields to be removed/excluded from a document         | 
```

**Review by Fashander — COMMENTED** [review]
Looks Good. 

We need to leave a new line at the end of every file. See here for information on formatting and building documentation https://github.com/FerretDB/FerretDB/blob/main/CONTRIBUTING.md#contributing-documentation

File: website/docs/aggregation-operations/aggregation-stages.md [hunk 3]
```diff
@@ -12,6 +12,8 @@ Aggregation stages are a series of one or more processes in a pipeline that acts
 | `$group`                     | Groups documents based on specific value or expression and returns a single document for each group   |
 | `$limit`                     | Limits specific documents and passes the rest to the next stage                                       |
 | `$match`                     | Acts as a `find` operation by only returning documents that match a specified query to the next stage |
+| `$project` | Specifies `n` number of fields in a document to pass to the next stage in the pipeline                               |
 | `$skip`                      | Skips a specified `n` number of documents and passes the rest to the next stage                       |
 | `$sort`                      | Sorts and returns all the documents based on a specified order                                        |
+| `$unset` | Specifies `n` number of fields to be removed/excluded from a document         | 
 | `$unwind`                    | Deconstructs and returns a document for every element in an array field                               |
```
Comment (Fashander) [inline website/docs/aggregation-operations/aggregation-stages.md:19]:
```suggestion
| `$unwind`                    | Deconstructs and returns a document for every element in an array field                               |

```

File: website/docs/aggregation-operations/aggregation-stages.md [hunk 4]
```diff
@@ -12,6 +12,9 @@ Aggregation stages are a series of one or more processes in a pipeline that acts
...
+
```
Comment (Fashander) [inline website/docs/aggregation-operations/aggregation-stages.md:20]:
```suggestion
```

**Comment by Akhaled19** [comment]
Hi, I resolved all the comments. Is it ready to be merged?

**Review by Fashander — DISMISSED** [review]

**Review by AlekSi — COMMENTED** [review]
Documentation should be reformatted with `task docs-fmt` because CI fails: https://github.com/FerretDB/FerretDB/actions/runs/6382671742/job/17321772162?pr=3450

**Review by AlekSi — APPROVED** [review]

**Review by Fashander — APPROVED** [review]

**Pull request merged**